app/views/staff/program_sessions/_form.html.haml

```diff
@@ -1,43 +1,29 @@
-= simple_form_for proposal, url: [ event, :staff, proposal ] do |f|
+= simple_form_for @program_session, url: [ event, :staff, @program_session ] do |f|
   .row
     %fieldset.col-md-6
-      %h4 Proposal
-      = proposal.title_input(f)
-      = proposal.abstract_input(f)
-      %section.inline-block
-        = f.label :video_url
-        = f.text_field :video_url, class: "form-control"
-      %section.inline-block
-        = f.label :slides_url
-        = f.text_field :slides_url, class: "form-control"
-
-  - if event.public_tags?
-    .row
+      = f.input :title, maxlength: 60, input_html: { class: 'watched js-maxlength-alert', rows: 1 }
+      = f.association :session_format, as: :select, collection: current_event.session_formats, label: "Format",
+        include_blank: false, required: true
+      = f.association :track, as: :select, collection: current_event.tracks, label: "Track"
+      - if !@program_session.new_record?
```

Comment:
I prefer unless  over if !.  The other piece is to use if @program_session.persisted? if you want to stick with an if instead of unless.